rclcpp/src/rclcpp/node_interfaces/node_base.cpp

```diff
@@ -104,6 +104,7 @@ NodeBase::NodeBase(
         }
         throw_from_rcl_error(RCL_RET_ERROR, "failed to validate node name");
       }
+      delete rcl_node;
```

Comment:
Bah, of course.  I missed all of the throw_from_*() functions.  I'll fix it, thanks.